go/externals/test_common.go

```diff
@@ -10,12 +10,14 @@ import (
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/pvlsource"
+	"github.com/keybase/client/go/uidmap"
 )
 
 func SetupTest(tb testing.TB, name string, depth int) (tc libkb.TestContext) {
 	ret := libkb.SetupTest(tb, name, depth+1)
 
 	ret.G.SetServices(GetServices())
+	ret.G.SetUIDMapper(uidmap.NewUIDMap(10000))
```

Comment:
@maxtaco this is the line that breaks TestUPAKDeadlock.  I can take it out, but then we aren't testing the uidmapper in lots of tests, we're using the test one that does not use api server endpoint at all.